Dockerfile-csi-controller.test

```diff
@@ -33,7 +33,7 @@ COPY ./controller /driver/controller
 USER root
 RUN groupadd -g 9999 appuser && \
```

Comment:
isn't better practice to write the group name in cap? (e.g. APPUSER)

Reply:
not that I'm aware of. but I didn't write this, I only chown another dir for some pylint pickle